generic/tests/netperf.py

```diff
@@ -482,6 +482,7 @@ def count_interrupt(name):
         sum = 0
         intr = []
         stat = ssh_cmd(server_ctl, "cat /proc/interrupts |grep %s" % name)
+        stat = stat.strip().split("\n")[-1]
```

Comment:
NACK, because the code would call this function as:
```python
513             nrx_intr = count_interrupt("virtio.-input")
```
So here should not only pick the `[-1]` one.

I guess the original problem you met is the command line itself was contained into output, you should check and remove it.